data/objects/library_controller.cfg

```diff
@@ -1265,18 +1265,18 @@
 			execute(me, [spawn(
 				'combo_controller',
 				{
-					x: gui_right_edge - py(710),
+					x: gui_right_edge - py(765),
 					y: level_height - py(110),
 					items: map(
 						lib.citadel.card_sets,
 						value.name),
 					selected_index:
 						requested_selected_index,
 					on_change: me._change_set,
-					_width: lib.gui.py(120),
-					_height: lib.gui.py(16),
+					_width: lib.gui.py(130),
+					_height: lib.gui.py(23),
 					zorder: 1000,
-					_font_size: lib.gui.py(12),
+					_font_size: lib.gui.py(20),
```

Comment:
I think this size change is good, but makes worse a problem that was there before your changes of this pull request. Can you revert this or else move it somewhere else in order for the list items to render inside the window?

Reply:
Hmm no, please don't mind this. My mistake.

Reply:
Yeah I see your point. This can be fixed by widening the combo_sets field. Also the font can be scaled down a bit. I'll push a commit with these changes.